tests/system/Autoloader/AutoloaderTest.php

```diff
@@ -286,6 +286,45 @@ public function testComposerNamespaceDoesNotOverwriteConfigAutoloadPsr4()
         $this->assertStringContainsString(VENDORPATH, $namespaces['Psr\Log'][1]);
     }
 
+    public function testComposerPackagesOnly()
+    {
+        $config                      = new Autoload();
+        $config->psr4                = [];
+        $modules                     = new Modules();
+        $modules->discoverInComposer = true;
+        $modules->composerPackages   = ['only' => ['laminas/laminas-escaper']];
+
+        $loader = new Autoloader();
+        $loader->initialize($config, $modules);
+
+        $namespaces = $loader->getNamespace();
+
+        $this->assertCount(1, $namespaces);
+        $this->assertStringContainsString(VENDORPATH, $namespaces['Laminas\Escaper'][0]);
+    }
+
+    public function testComposerPackagesExlcude()
```

Comment:
If you stick with the "only/exclude" Config style then please add another test for both:
```php
    public function testComposerPackagesOnlyAndExlcude()
```

Reply:
Added.

Reply:
Thanks! Good exception.